<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>481</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>文本接收</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>原始接收文本</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>解调后文本</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>解码后文本</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="textBrowser_original"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="textBrowser_demodulated"/>
    </item>
    <item row="1" column="2">
     <widget class="QTextBrowser" name="textBrowser_decoded"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_sample_rate">
      <property name="text">
       <string>采样率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>276</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>客户端操作</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口号: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextBrowser" name="textBrowser_client_info"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btn_connect">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>建立连接</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_ip"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP地址: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>文件操作</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="btn_load_received_file">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>加载已接收的原始文件</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_demodulate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>开始解调</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_demodulation">
      <item>
       <property name="text">
        <string>ASK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSK</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_decode">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>开始解码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_decoding">
      <item>
       <property name="text">
        <string>UTF-8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UTF-16</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="btn_save_demodulated_file">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>保存恢复后的文本文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
